test/powershell/Language/Scripting/Redirection.Tests.ps1

```diff
@@ -0,0 +1,10 @@
+
+Describe "File redirection" -Tag CI {
```

Comment:
Nope, I missed that file.  Maybe the parser and script directories should be merged?